RecoVertex/BeamSpotProducer/plugins/BeamSpotOnlineProducer.cc

```diff
@@ -1,10 +1,47 @@
+/**_________________________________________________________________
+   class:   BeamSpotOnlineProducer.h
+   package: RecoVertex/BeamSpotProducer
 
-#include "RecoVertex/BeamSpotProducer/interface/BeamSpotOnlineProducer.h"
-#include "DataFormats/BeamSpot/interface/BeamSpot.h"
 
+
+ author: Francisco Yumiceva, Fermilab (yumiceva@fnal.gov)
+
+
+________________________________________________________________**/
+
+#include "DataFormats/BeamSpot/interface/BeamSpot.h"
+#include "DataFormats/Common/interface/Handle.h"
+#include "DataFormats/L1GlobalTrigger/interface/L1GlobalTriggerEvmReadoutRecord.h"
+#include "DataFormats/Scalers/interface/BeamSpotOnline.h"
+#include "CondFormats/BeamSpotObjects/interface/BeamSpotObjects.h"
+#include "CondFormats/DataRecord/interface/BeamSpotObjectsRcd.h"
+#include "FWCore/Framework/interface/ESHandle.h"
+#include "FWCore/Framework/interface/Event.h"
 #include "FWCore/Framework/interface/EventSetup.h"
-#include "FWCore/Framework/interface/IOVSyncValue.h"
-#include "FWCore/Framework/interface/MakerMacros.h"
+//#include "FWCore/Framework/interface/IOVSyncValue.h"
```

Comment:
is this commented out code needed?
Please remove or add comments inline in the code why the commented out block is relevant

Reply:
... this is new compared to the previously signed version by reco